website/static/js/fangorn.js

```diff
@@ -457,7 +466,17 @@ function checkConflictsRename(tb, item, name, cb) {
     for(var i = 0; i < parent.children.length; i++) {
         var child = parent.children[i];
         if (child.data.name === name && child.id !== item.id) {
-            messageArray.push(m('p', 'An item named "' + child.data.name + '" already exists in this location.'));
+            messageArray.push([
+                m('p', 'An item named "' + child.data.name + '" already exists in this location.'),
+                m('h5.text-danger.replace-file',
+                    '"Keep Both" will retain both files (and their version histories) in this location.'),
+                m('h5.text-danger.replace-file',
+                    '"Replace" will overwrite the existing file in this location. ' +
+                    'You will lose previous versions versions of the overwritten file. ' +
```

Comment:
Still need to fix this typo in two spots: 'You will lose previous **versions versions** of the overwritten file. '